src/Git/CreateTagViaConsole.php

```diff
@@ -6,25 +6,39 @@
 
 use Laminas\AutomaticReleases\Git\Value\BranchName;
 use Laminas\AutomaticReleases\Gpg\SecretKeyId;
-use Psl\Env;
-use Psl\File;
-use Psl\Filesystem;
-use Psl\Shell;
+
+use function Psl\Env\temp_dir;
+use function Psl\File\write;
+use function Psl\Filesystem\create_temporary_file;
+use function Psl\Shell\execute;
```

Comment:
Noticed this on other patches of yours: probably done by the IDE, but consider not importing single functions, but just their namespace.

It is much much easier to identify `Shell\execute()` in the middle of the code, rather than `execute()`. This becomes much more visible when working with `Psl\Type\*`